src/constants/types.d.ts

```diff
@@ -30,3 +30,5 @@ export enum AuthType {
 export interface ModalPageProps {
   onClose?: () => void;
 }
+
+export const DefaultStellarBipPath = "44'/148'/0'";
```

Comment:
Would `settings` be a better place for this const? It's not really a type. 🤔  + all caps or starting with a lower `defaultStellarBipPath`.